templates/Alloy.Mvc/Views/CustomContentAreaItemCssRenderingBase.cs

```diff
@@ -0,0 +1,68 @@
+using EPiServer.ServiceLocation;
+using EPiServer.Web;
+using Microsoft.AspNetCore.Mvc.TagHelpers;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+using Microsoft.AspNetCore.Mvc.Razor;
+using static Alloy.Mvc._1.Globals;
+using Alloy.Mvc._1.Business.Rendering;
+using System.Text;
+
+namespace Alloy.Mvc._1.Views;
+
+public abstract class CustomContentAreaItemCssRenderingBase<TModel> : RazorPage<TModel> where TModel : class
```

Comment:
The name is descriptive, which is great. But it's a long and handful name to inherit from. Shouldn't this class inherit `RazorPageModel` to get the routed content?

Reply:
I have refactored the business to a server and change the base class name. The class could not inherit from `RazorPageMode` because it accept a view model instead of an `IContent`.